courses/machine_learning/deepdive2/recommendation_systems/labs/als_bqml.ipynb

```diff
@@ -0,0 +1,1399 @@
+{
```

Comment:
<p>Maybe since it is the lab version, just clear outputs as we normally do for lab versions and then re-save.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/753/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>